cmd/keeper/cmd/keeper.go

```diff
@@ -811,18 +813,50 @@ func (p *PostgresKeeper) resync(db, followedDB *cluster.DB, tryPgrewind bool) er
 	// doesn't exists pgm.SyncFromFollowedPGRewind will return an error and
```

Comment:
Should we maybe modify this comment? Or add another?

Reply:
This comment is still legit so I think we should leave it as is.

Is there something else you think we also need to add here?

Reply:
We just need to add a bit about the retry. If we have retry setup, we won't fallback to `pg_basebackup` right away as this comment suggests.